Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16
+
+MAINTAINER Ionuț Roșca <ionut.rosca@info.uaic.ro>
+LABEL version="0.1.1"
+
+# Create the app's folder on the container
+WORKDIR /usr/share/src/app
+
+# Ensure that both package.json and package-lock.json are copied
+COPY package*.json ./
+
+RUN npm install
```

Comment:
The package lock shouldn't be a must. That's generated by installing the modules. If you need the package lock, then do the npm install first

Reply:
From where I've found the tutorial to create the `Dockerfile` for a Node.js app, it was in that order. And I have also found an answer if `package-lock.json` is needed [here](https://stackoverflow.com/a/65234529/10604490).

And because of the findings I found regarding the `package-lock.json` file, I will replace `npm install` with `npm ci`, which is meant to be used in production evironments, continuous integration environments etc. in favor of `npm install`. And from [npm-ci docs](https://docs.npmjs.com/cli/v7/commands/npm-ci#description):

> The project must have an existing package-lock.json or npm-shrinkwrap.json.